### DISCUSSION TITLE
Problem fifa 2000 on vmware windows 98

### DISCUSSION THREAD

**Discussion by win32forever** [JHRobotics/softgpu#97] [General]
Hello fantastic team I have a problem, FIFA 2000 does not appear in the gameplay and it crashes immediately after loading with the controller screen

**Comment by JHRobotics** [comment]
Hi, you have to switch renderer to DirectX -> In game directory is "3DSetup" folder and in there is "D3Setup.exe" program. Run it and set *3D Graphic Renderer* to **D3D Device**. The message, that video card is not known can be safely ignore :-)

![fifa2000_d3d](https://github.com/user-attachments/assets/4b7ac7ca-d7fa-418c-86e3-e423c135a1a2)

Also, if you want larger resolution than 640x480, open regedit and navigate to `HKEY_LOCAL_MACHINE\Software\EA SPORTS\FIFA 2000` and set `Trash Resolution` to something larget. When you do it, menu is still remain in 640x480 but when you start match, game should switch to this resolution.

![fifa2000_1024x768](https://github.com/user-attachments/assets/eadbe127-bce1-47d8-9550-f47413a912c9)

**Reply by win32forever** [reply]
thank you very much, I managed to get it started only that the field is completely black
